{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_tasks.py

```diff
@@ -9,8 +9,16 @@
 
 def test_user_count(settings):
     """A basic test to execute the get_users_count Celery task."""
-    UserFactory.create_batch(3)
     settings.CELERY_TASK_ALWAYS_EAGER = True
-    task_result = get_users_count.delay()
-    assert isinstance(task_result, EagerResult)
-    assert task_result.result == 3
+
+    # Get all existing users in the DB
+    current_users = get_users_count.delay()
+    assert isinstance(current_users, EagerResult)
```

Comment:
Any reason to use the task to get the previous cont instead of calling directly `User.objects.count()`?

Might be clearer if it was:

```python
previous_count = User.objects.count()
UserFactory.create_batch(3)
... (same as old code)
assert isinstance(task_result, EagerResult)
assert task_result.result == previous_count + 3
```

Reply:
@browniebroke @Andrew-Chen-Wang Thank you for your comments. I certainly was confused. Reading the [link](https://docs.djangoproject.com/en/3.1/topics/testing/overview/#the-test-database) regarding how Django creates a test DB cleared it up.

> Any migrations will also be applied in order to keep it up to date.


The tests on my local dev were failing and I couldn't explain them and came to this incorrect conclusion that my DB is being re-used when in fact it was due to me creating a superuser using migrations! 

Thanks for the help. 

@browniebroke Regarding your code review comment, I just wanted to use celery to get the number of users. No other reason


This PR seems unnecessary now although if someone like me ends up using migrations to add users then they will get the same issue. Let me know if I should just close this issue.

Reply:
Using Celery to get the count of users before and after seems to overcomplicate the test. I'd rather keep it simple.

The point of this test is just to have something testing that Celery is configured correctly. It's mainly for the sake of Cookiecutter Django, more than the projects you generate.

I'll leave it up to you if you want to simplify it or close the PR.

Thanks.